example_routing.md

```diff
@@ -0,0 +1,242 @@
+
+# Simple routing spike
+
+Below I've set out some options for simple routing data structuring, and a bunch of other stuff. There two main options
+
+## Option 1
+
+Questions either have a next page when they're unconditional, or a set of conditions if there need to be conditional routing checks. This is not as preferable as the second option, as it requires us to construct more of a branching system whereby all outputs to a conditional question are specified - so all routes out of that question are covered by conditions. The user will implicitly set up routes in different directions, and we'd have to calculate them explicitly. 
+
+There's potentially a bit of value down the line in being extremely explicit about all the ways a route resolves, but it's kind of hard to think about them now.
+
+### api
+```json
+{
+  id: 1,
+  next_page: 2,
+  conditions: nil,
```

Comment:
Very minor syntax thing but none of these JSON snippets are technically valid - keys need to be strings and `nil` is Ruby:
```suggestion
  "id": 1,
  "next_page": 2,
  "conditions": null,
```

Reply:
Ah good spot, no wonder my editor was flagging it as nonsense - I'll tidy it up a bit